### ISSUE TITLE
Dropdown: The down arrow doesn't have an accessible text

### ISSUE THREAD

**Issue by JaneSjs** [surveyjs/survey-library#7259]
User Issue: T15286 - Accessibility test
https://surveyjs.answerdesk.io/internal/ticket/details/T15286
***
Enable the Accessibility Insights for Web and Run the Fast Pass check over [a Dropdown question](https://surveyjs.io/form-library/examples/create-dropdown-menu-in-javascript/reactjs). The following issue appears:
***
Title: WCAG 1.1.1: Ensures <svg> elements with an img, graphics-document or graphics-symbol role have an accessible text (.sd-dropdown_chevron-button-svg)
Tags: Accessibility, WCAG 1.1.1, svg-img-alt

Issue: Ensures <svg> elements with an img, graphics-document or graphics-symbol role have an accessible text (svg-img-alt - https://dequeuniversity.com/rules/axe/4.7/svg-img-alt?application=msftAI)

Target application: https://surveyjs.io/form-library/examples/s/create-dropdown-menu-in-javascript/reactjs/default

Element path: .sd-dropdown_chevron-button-svg

Snippet: <svg class="sv-svg-icon sd-dropdown_chevron-button-svg" role="img"><use xlink:href="#icon-chevron"></use></svg>

How to fix: 
Fix any of the following:
  Element has no child that is a title
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

Environment: Chrome version 118.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.40.0 (axe-core 4.7.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
